### ISSUE TITLE
Reduce false positives in node link report.

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#1524]
Resolve the following links 
- [ ] https://veteran.mobile.va.gov/aap-web/#PillBottle 
This one is caused by the site returning a 304 (nothing modified) which is likely because it is a mobile app that limits tranfer of data if nothing changed.   This is a case where the module should treat a 304 like a 200.  https://www.drupal.org/project/node_link_report/issues/3128744

Domains that are blocked by the VA network that should just be added to the exempted domain config.
- [x] www.choicehotels.com
- [x] www.hilton.com
- [x] www.marriott.com
- [x] hamptoninn3.hilton.com
- [x] hiltongardeninn3.hilton.com
- [x] www.qualityinn.com
- [x] www.portauthority.org/

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information if relevant, or delete):**

**Additional context**

## Labels
- [/] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [/] CMS objective (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)
